mapviz_plugins/src/gps_plugin.cpp

```diff
@@ -39,17 +39,12 @@
 
 // Declare plugin
 #include <pluginlib/class_list_macros.h>
-PLUGINLIB_DECLARE_CLASS(
-    mapviz_plugins,
-    gps,
-    mapviz_plugins::GpsPlugin,
-    mapviz::MapvizPlugin);
+PLUGINLIB_DECLARE_CLASS(mapviz_plugins, gps, mapviz_plugins::GpsPlugin,
+                        mapviz::MapvizPlugin);
```

Comment:
This change makes the arguments to this macro much harder to read. If it's the indentation you don't like, then restyling like this would also work:
```
PLUGINLIB_DECLARE_CLASS(mapviz_plugins,
                        gps,
                        mapviz_plugins::GpsPlugin,
                        mapviz::MapvizPlugin);
```
Every `_plugin.cpp` has this call, and they should be consistent, so if you fix one, make sure to fix them all.